plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/vmware/resource/VmwareResource.java

```diff
@@ -2068,13 +2068,16 @@ protected StartAnswer execute(StartCommand cmd) {
                         }
                     }
                 } else {
-                    controllerKey = vmMo.getScsiDiskControllerKeyNoException(diskController);
+                    if (VmwareHelper.isReservedScsiDeviceNumber(scsiUnitNumber))
```

Comment:
Minor one: can you please add braces to this if statement?

Reply:
> Minor one: can you please add braces to this if statement?

updated this